api/jwt/token_blacklist.go

```diff
@@ -0,0 +1,86 @@
+package jwt
```

Comment:
Naming:

The files are still include `blacklist`. Would you mind and rename the files too. That'd be lovely :+1: